tests/runtest.proj

```diff
@@ -118,7 +121,7 @@ $(_XunitEpilog)
   </ItemGroup>
   <Import Project="$(SourceDir)dir.targets" />
   <PropertyGroup>
-     <OutDir>$(XunitTestBinBase)\$(Category)\</OutDir>
+     <OutDir>$(XunitTestBinBase)\$(CategoryWithSlash)\</OutDir>
```

Comment:
I find this name confusing since there was a slash following it before, now it is "WithSlash" but still has a trailing slash.